packages/sdk/src/actions/executeQuote.ts

```diff
@@ -281,6 +285,109 @@ export async function executeQuote(params: ExecuteQuoteParams): Promise<ExecuteQ
 
         onProgressHandler(currentTransactionProgress);
 
+        // Check if wallet supports atomic transactions
```

Comment:
It would be nice to have a flag that disables the flow via `getCapabilities`, something like `disableWalletCapabilites`. Background is that we have some e2e test that use local Anvil forks which don't seem to support `wallet_*` RPC requests. See [here](https://github.com/across-protocol/toolkit/actions/runs/15749730963/job/44392535326?pr=229).

So we could use this flag to not use the capabilities flow in the tests.

Reply:
Thanks for flagging, can definitely make it opt in but i might also try to make it so we can test e2e, i think we could take a similar approach to viem's unit test for sendCalls https://github.com/wevm/viem/blob/feat%2Fcalls-data-suffix/src%2Factions%2Fwallet%2FsendCalls.test.ts

Reply:
Will take a look at viem's test setup as well. Thanks for flagging this